helm/aws-efs-csi-driver/values.yaml

```diff
@@ -83,3 +83,27 @@ serviceAccount:
 
 controller:
   create: false
+  # Add additional tags to access points
+  tags:
+    {}
+    # environment: prod
+    # region: us-east-1
+  # Enable if you want the controller to also delete the
+  # path on efs when deleteing an access point
+  deleteAccessPointRootDir: false
```

Comment:
it's by default set to false however it's possible when controller is enabled to change it.